pages/index.js

```diff
@@ -6,6 +6,7 @@ import Header from "../components/Header";
 import CreateNewForm from "../components/Form/AddForm";
 
 import Card from "../components/Form/Cards";
+import { nanoid } from "nanoid";
 
 export default function Home() {
   const [cardData, setCardData] = useState(fakeHabitDB);
```

Comment:
this sounds like it is the data for a single card, you can name it habitsList or habitsData

Reply:
a basic rule of thumb: arrays are named plural, individual elements are named in singular, boolean names indicate a true or false statement.